packages/docs/components/layout/header/HeaderSelector.vue

```diff
@@ -39,7 +39,7 @@ export default defineComponent({
   props: {
     minimized: {
       type: Boolean,
-      required: true,
+      default: false,
```

Comment:
I think we 100% need this to be required, so we handle if sibebar can be toggled. It will be a bug to have this always default to false. If you want this flexable, you can change default to `undefined` and hide the whole button. We don't need this level of flexibility, so this prop must be required.